templates/bake/Command/command.twig

```diff
@@ -13,15 +13,18 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 #}
-<?php
-declare(strict_types=1);
-
-namespace {{ namespace }}\Command;
-
-use Cake\Command\Command;
-use Cake\Console\Arguments;
-use Cake\Console\ConsoleIo;
-use Cake\Console\ConsoleOptionParser;
+{{ element('Bake.file_header', {
+        namespace: "#{namespace}\\Command",
+        uses: {
+            class: [
+                'use Cake\\Command\\Command;',
```

Comment:
Having to prefix `use ` for all the classes seems a bit redundant.

Reply:
This ended up being the final iteration, but I tried others.

To avoid having to do awkward code-gen in the template, FileBuilder returns the code just like ClassBuilder does.

The class, function and const uses all have different prefixes, would you have had to code gen them separately.

I also need to allow users and non-model templates to add imports without the help of FileBuilder, so it ended up being arrays of staetments.

I could make them just lists of imports, but that requires a special array format and code-gen. It would also mean FileBuilder wouldn't return code.

Reply:
@markstory We should resolve this before beta

Reply:
Whether it's a class, function or const the `use` keyword will be the same for all. The `file_header` template could add that prefix instead of having to repeat it for each class/function/constant.

Reply:
That feels hacky. Then we'd have `function import` as the parameter which is neither code nor import.

We'd have to move back to arrays of imports/aliases.